wtforms_components/fields/json_field.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from wtforms import fields, widgets
+
+
+class JSONField(fields.StringField):
+    """
+    A text field which stores a `json`.
+    """
+    widget = widgets.TextArea()
+
+    def __init__(self, label=None, validators=None, **kwargs):
```

Comment:
You don't have to define the `__init__` method. You can remove it completely.

Reply:
Removed. Thanks.